docs/Outbound_Vulnerability_Disclosure_Policy_template.md

```diff
@@ -14,18 +14,18 @@ The OpenSSF Vulnerability Disclosure WG Autofix SIG is working on an update to t
 
 Open an issue under the [OpenSSF Vulnerability Disclosure Working Group Repository](https://github.com/ossf/wg-vulnerability-disclosures/issues), or ask in the [OpenSSF Slack](https://slack.openssf.org/) under the WG Vulnerability Disclosure channel.
 
-# Model Outbound Vulnerability Disclosure Policy: Version 0.1
+# Model Outbound Vulnerability Disclosure Policy: Version 0.2
 
 ## Manual Disclosure Policy
 
-We believe that vulnerability disclosure is a collaborative, two-way street. All parties involved, including but not limited to maintainers[^1], as well as researchers, must act responsibly. This is why we adhere to a maximum 90-day public disclosure time limit, the “Time Limit”. We immediately privately report to maintainers when we discover vulnerabilities within their software, the “Notice Date”. If a project responds to the private report within 21 calendar days, the details will be publicly disclosed (shared with the defensive community)  90 days after the Notice Date, on the “Publication Date”, or sooner if the maintainer releases a fix prior to the Publication Date. That Publication Date can vary in the following ways:
+We believe that vulnerability disclosure is a collaborative, two-way street. All parties, maintainers[^1], as well as researchers, must act responsibly. This is why we adhere to a maximum 90-day public disclosure time limit, the “Time Limit”. We immediately privately report to maintainers when we discover vulnerabilities within their software, the “Notice Date”. If a project responds to the private report within 21 calendar days, the details will be publicly disclosed (shared with the defensive community) after 90 days, the “Publication Date”, or sooner if the maintainer releases a fix prior to the Publication Date. That Publication Date can vary in the following ways:
```

Comment:
hmm, removing `after the Notice Date`, seems to ambiguate this.  what is the purpose of that change?

Reply:
Huh, the starting version, on GitHub, seems to have deviated from the version we originally copied from, which was this version on the website:

https://openssf.org/about/vulnerability-disclosure-policy/

Bah, we've already got version drift even within the OpenSSF! 😆 

We should stay with the more explicit version IMHO

Reply:
I'll restore this to the original. I've added an internal tracking task for myself to ensure that the OpenSSF website gets updated to reflect this change